src/Microsoft.Data.Analysis/DataFrame.cs

```diff
@@ -623,12 +623,16 @@ private void OnColumnsChanged()
         private DataFrame Sort(string columnName, bool isAscending)
         {
             DataFrameColumn column = Columns[columnName];
-            DataFrameColumn sortIndices = column.GetAscendingSortIndices();
+            PrimitiveDataFrameColumn<long> sortIndices = column.GetAscendingSortIndices(out Int64DataFrameColumn nullIndices);
+            for (long i = 0; i < nullIndices.Length; i++)
+            {
+                sortIndices.Append(nullIndices[i]);
```

Comment:
This would cause `null` rows to be at the top for a descending sort. I have to think about if this is acceptable or if we want to change this behavior. Just calling it out here

Reply:
For the moment, I think we can leave the null rows at the top, until we get feedback about it (if any). To move the null rows to the bottom, we'd need to thread the `nullIndices` column into the `Clone` API implementations, and I don't think it's worth it currently.

Reply:
Looks like Pandas puts them in the 'last' position by default, but you can change it with an option.